src/libFLAC/stream_decoder.c

```diff
@@ -2123,13 +2123,15 @@ FLAC__bool read_frame_(FLAC__StreamDecoder *decoder, FLAC__bool *got_a_frame, FL
 		}
 	}
 	else {
-		/* Bad frame, emit error and zero the output signal */
+		/* Bad frame, emit error, zero the output signal, return */
 		send_error_to_client_(decoder, FLAC__STREAM_DECODER_ERROR_STATUS_FRAME_CRC_MISMATCH);
```

Comment:
Can you clarify (perhaps in commit message) why this should be kept? Some of the other early exits do not seem to do that.